packages/pc/src/behavior/brush-select.ts

```diff
@@ -170,16 +177,39 @@ export default {
         });
       });
     }
+    const selectedCombos = [];
+    if (this.includeCombos) {
+      graph.getCombos().forEach((combo) => {
+        if (!combo.isVisible()) return;
+        const bbox = combo.getBBox();
+        if (
+          bbox.centerX >= left &&
+          bbox.centerX <= right &&
+          bbox.centerY >= top &&
+          bbox.centerY <= bottom
+        ) {
```

Comment:
I wonder how often we have to do an operation like this? It's not very nice to read.... If there are multiple areas where we do this maybe we could make a util function `bboxInbounds(centerX, centerY) => bool` ?

Reply:
Hmm... we _may_ also have to do it for edges at some point. Current code "includes" edges when the _source_ node of an edge is included in the selection, which different from what KL does - that goes by midpoint, same check in this `if`.

So yes it makes sense, I'll add it.